packages/web-ui-registration/src/LoginForm.tsx

```diff
@@ -0,0 +1,179 @@
+						setError('email', { type: 'invalid-email', message: t('Invalid_email') });
+					}
+
+					if ('error' in error && error.error !== 403) {
+						setErrorOnSubmit(error.error);
+						return;
+					}
+
+					setErrorOnSubmit('user-not-found');
+					setError('username', { type: 'user-not-found', message: t('User_not_found') });
+					setError('password', { type: 'user-not-found', message: t('User_not_found') });
+				}
+			})}
+		>
+			<Form.Header>
+				<Form.Title id={formLabelId}>{t('registration.component.login')}</Form.Title>
+			</Form.Header>
+			<Form.Container>
+				<Services />
+				<FieldGroup>
+					<Field>
+						<Field.Label htmlFor='username'>{t('registration.component.form.emailOrUsername')}</Field.Label>
+						<Field.Row>
+							<TextInput
+								{...register('username', {
+									required: true,
+								})}
+								placeholder={t('registration.component.form.emailOrUsernamePlaceholder')}
+								error={
+									errors.username?.message ||
+									(errors.username?.type === 'required' ? t('registration.component.form.requiredField') : undefined)
+								}
+								aria-invalid={errors.username || errorOnSubmit === 'user-not-found' ? 'true' : 'false'}
+								id='username'
+							/>
+						</Field.Row>
+						{errors.username && errors.username.type === 'required' && (
+							<Field.Error>{t('registration.component.form.requiredField')}</Field.Error>
+						)}
+					</Field>
+
+					<Field>
+						<Field.Label htmlFor='password'>{t('registration.component.form.password')}</Field.Label>
+						<Field.Row>
+							<PasswordInput
+								{...register('password', {
+									required: true,
+								})}
+								placeholder={'*****'}
+								error={
+									errors.password?.message ||
+									(errors.password?.type === 'required' ? t('registration.component.form.requiredField') : undefined)
+								}
+								aria-invalid={errors.password || errorOnSubmit === 'user-not-found' ? 'true' : 'false'}
+								id='password'
+							/>
+						</Field.Row>
+						{errors.password && errors.password.type === 'required' && (
+							<Field.Error>{t('registration.component.form.requiredField')}</Field.Error>
+						)}
+						{isResetPasswordAllowed && (
+							<Field.Row justifyContent='end'>
+								<Field.Link
+									href='#'
+									onClick={(e): void => {
+										e.preventDefault();
+										setLoginRoute('reset-password');
+									}}
+								>
+									<Trans i18nKey='registration.page.login.forgot'>Forgot your password?</Trans>
+								</Field.Link>
+							</Field.Row>
+						)}
+					</Field>
+				</FieldGroup>
+				<FieldGroup>
+					{errorOnSubmit === 'error-user-is-not-activated' && (
+						<Callout type='warning'>{t('registration.page.registration.waitActivationWarning')}</Callout>
+					)}
+
+					{errorOnSubmit === 'error-app-user-is-not-allowed-to-login' && (
+						<Callout type='danger'>{t('registration.page.login.errors.AppUserNotAllowedToLogin')}</Callout>
```

Comment:
why `Callout` instead of `ToastBar`?

Reply:
this is how was implemented before maybe is to fast to a regular user read, and is a important information